src/formatter.js

```diff
@@ -79,23 +79,31 @@ module.exports.getType = (input) => {
  *
  * @return momentJS
  **/
-module.exports.parse = (input, type) => {
+module.exports.parse = (input, type, isUTC) => {
+    let result;
+
     switch(type) {
         case INPUT_FORMAT.NOW:
-            return moment()
+            result = moment()
             break
         case INPUT_FORMAT.TS_S:
-            return moment.unix(parseInt(input))
+            result = moment.unix(parseInt(input))
             break
         case INPUT_FORMAT.TS_MS:
-            return moment.unix(parseInt(input) / 1000)
+            result = moment.unix(parseInt(input) / 1000)
```

Comment:
this needs to be done for nanoseconds as well

Reply:
#5 has been merged, you can now do it for nanoseconds.

Thanks!

Reply:
Done now :+1: